tests/test_kitti_eval.py

```diff
@@ -0,0 +1,243 @@
+import numpy as np
+import pytest
+import torch
+
+from mmdet3d.core.evaluation.kitti_utils.eval import (do_eval, eval_class,
+                                                      kitti_eval)
+
+
+def test_do_eval():
+    if not torch.cuda.is_available():
+        pytest.skip('test requires GPU and torch+cuda')
```

Comment:
1. Check whether the evaluation could be performed without GPU.
2. Change the message, `test requires GPU and CUDA` is more precise.